plugins/goodix-fp/goodixfp.quirk

```diff
@@ -0,0 +1,9 @@
+# Goodix Fingerprint sensor
+[DeviceInstanceId=USB\VID_27C6&PID_60A2]
+Plugin = goodixfp
+Guid = 1542f422-f06b-5c0c-8211-30c102a65f01
+Flags = self-recovery
```

Comment:
Just to make sure this flag and only this flag makes sense here - this means that  "In flashing mode device will only accept intended payloads".   You can double check the other flags here to see if any others should apply: https://github.com/fwupd/fwupd/blob/master/libfwupd/fwupd-enums.h#L86